src/library-authoring/component-info/ComponentUsageTab.tsx

```diff
@@ -0,0 +1,14 @@
+import { useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+import { ComponentHierarchy } from '../hierarchy/ComponentHierarchy';
+
+export const ComponentUsageTab = () => {
+  const intl = useIntl();
+
+  return (
+    <>
+      <h4>{intl.formatMessage(messages.usageTabHierarchyHeading)}</h4>
+      <ComponentHierarchy />
+    </>
+  );
+};
```

Comment:
Nit: We can make it compact by using the `FormattedMessage` component
```suggestion
import { FormattedMessage } from '@edx/frontend-platform/i18n';
import messages from './messages';
import { ComponentHierarchy } from '../hierarchy/ComponentHierarchy';

export const ComponentUsageTab = () => (
  <>
    <h4><FormattedMessage {...messages.usageTabHierarchyHeading} /></h4>
    <ComponentHierarchy />
  </>
};
```

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/2399/commits/72e3608b851d7019c3c9883512bde0a920956248